modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -32,6 +32,7 @@ config PM_PARTITION_SIZE_MCUBOOT
 config PM_PARTITION_SIZE_MCUBOOT
 	hex
```

Comment:
please add prompt to allow changing that value by user (add prompt also in line 28)